### PR TITLE
Stats counter PR

### PR DESCRIPTION
**Pull request by ichala**
In this PR i do the following : 
- Add all items counter on the Homepage

### CODE REVIEW

File: src/modules/Functions.js [hunk 1]
```diff
@@ -96,7 +106,18 @@ function DisplayCards(data) {
...
+function Counter(data) {
+//   const LikesCounter = document.querySelectorAll('.likes-counter');
+  ProductsCounter.innerHTML = data.products.length;
+  // data.likes.forEach(item => {
+  //     LikesCounter.forEach(counter=>{
+  //         if(item.item_id === counter.id) {
+  //             counter.innerHTML=item.likes
+  //         }
+  //     })
+  // })
```
Comment (kwambiee) [inline src/modules/Functions.js:120]: Kindly, remove the comments to make your work much cleaner.

File: src/modules/Functions.js [hunk 2]
```diff
@@ -96,7 +106,18 @@ function DisplayCards(data) {
...
+}
```
Comment (kwambiee) [inline src/modules/Functions.js:121]: Kindly, remove the comments to make your work much cleaner. #27

**Review by kwambiee — CHANGES_REQUESTED** [review]
Hi @ichala , 

Good job so far!
There are some issues that you still need to work on to make sure that the code is following the best practices

the-simpsons-homer-simpson

## To Highlight 🎉

- No linter errors. ✔️
- Great use of the api. ✔️
- Great file organization. ✔️
- PR has a good title and summary. ✔️

 ## Required Changes 

- [ ] ♻️Check comment line below

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/Functions.js (hunk 2 above)
Comment (ichala) [reply src/modules/Functions.js:113]: Done

**Pull request merged**